### DISCUSSION TITLE
we got 4791 plot files and got 0 coins

### DISCUSSION THREAD

**Discussion by chiafuns** [Chia-Network/chia-blockchain#2815] [Q&A]
hi all,
       we have finished 4000 more plot files by many computers together(use the same fingerprint).
       we  got 1754 plot files and got 2 coins in 2021/4/22
       we  got 2376 plot files and got 2 coins in 2021/4/23
        we got 2966 plot files and got 0 coins in  2021/4/24
        we got 3312 plot files and got 0 coins in 2021/4/25
         we got 4791 plot files and got 0 coins in 2021/4/26
         what's the problem with me? could you help me fix the problem? Thanks!

**Comment by chiafuns** [comment]
this is one computer's screenshot. 
![image](https://user-images.githubusercontent.com/83232426/116112937-a1ad9d80-a6ea-11eb-8d1a-0e692d348ef8.png)

**Reply by dorkmo** [reply]
are there more plots than the 269 listed on this computer? you might need to add the directory on the hard drives with the plots.

**Reply by dorkmo** [reply]
go to the plots tab, the click the button with the 3 dots in the upper right corner, then select "add plot directory"

**Comment by chiafuns** [comment]
we have 20 computer together to plot the file(use the same account number)
Is it normal？

**Comment by chiafuns** [comment]
we have 20 computers!
this is just one computer's plots count!

---Original---
From: ***@***.***&gt;
Date: Tue, Apr 27, 2021 08:32 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Chia-Network/chia-blockchain] we got 4791 plot files and got 0 coins (#2815)




 
are there more plots than the 269 listed on this computer? you might need to add the directory on the hard drives with the plots.

**Comment by chiafuns** [comment]
k32 101g plot file!



---Original---
From: "Dima ***@***.***&gt;
Date: Tue, Apr 27, 2021 06:17 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Chia-Network/chia-blockchain] we got 4791 plot files and got 0 coins (#2815)




 
@chiafuns k=?
 min k=32

**Comment by chiafuns** [comment]
we have 20 computers! we think we didn't add all the plot directorys to one computer!



---Original---
From: ***@***.***&gt;
Date: Tue, Apr 27, 2021 08:33 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Chia-Network/chia-blockchain] we got 4791 plot files and got 0 coins (#2815)




 
go to the plots tab, the click the button with the 3 dots in the upper right corner, then select "add plot directory"

**Reply by Crypto-78** [reply]
Are the 20 PCs all on the same network? Or did you follow the predura on Wiki "Farming on many machines"
